NuGet.config

```diff
@@ -1,9 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="OrchardCore" value="https://www.myget.org/F/orchardcore-dev/api/v3/index.json" />
+    <add key="Fluid" value="https://www.myget.org/F/fluid/api/v3/index.json" />
```

Comment:
Not needed, I added it as an upstream feed for `orchardcore-dev` which is referenced above

Reply:
Oh I thought I saw you do that, but it didn't work without it. I'll investigate.